practice5/src/test/java/com/academiaDigital/softwareArchitecture/practice5/builder/SpaceStationBuilderTest.java

```diff
@@ -0,0 +1,19 @@
+package com.academiaDigital.softwareArchitecture.practice5.builder;
+
+import com.academiaDigital.softwareArchitecture.practice5.products.SpaceStation;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class SpaceStationBuilderTest {
+
+    @Test
+    public void getSpaceStation() {

```

Comment:
in the majority of the cases we start each function name for testing with the `test` prefix, in this case, it'd be: `testSpaceStationGetter`

Reply:
Sounds good, on the other hand I have applied this convention <function name> should do or return <Error, empty, object>
then I changed according your suggestion.